wifiphisher/pywifiphisher.py

```diff
@@ -523,7 +523,7 @@ def start(self):
         except (interfaces.InvalidInterfaceError,
                 interfaces.InterfaceCantBeFoundError,
                 interfaces.InterfaceManagedByNetworkManagerError) as err:
-            logger.exception("Setting {} to monitor mode".format(mon_iface))
+            logger.exception("The following error has occurred:")
             print ("[{0}!{1}] {2}").format(R, W, err)
```

Comment:
The logging message doesn't print any reason for the error? The line below it is the stdout print.

Reply:
`logger.exception` automatically should put the exception in logs. Doesn't it?

Reply:
yes, correct. I misunderstood for this function. Thanks @blackHatMonkey : )